staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -205,14 +208,12 @@ func NewCmdApply(baseName string, f cmdutil.Factory, ioStreams genericclioptions
 }
 
 // Complete verifies if ApplyOptions are valid and without conflicts.
-func (o *ApplyOptions) Complete(f cmdutil.Factory, cmd *cobra.Command) error {
-	var err error
-	o.ServerSideApply = cmdutil.GetServerSideApplyFlag(cmd)
-	o.ForceConflicts = cmdutil.GetForceConflictsFlag(cmd)
-	o.DryRunStrategy, err = cmdutil.GetDryRunStrategy(cmd)
-	if err != nil {
-		return err
+func (o *ApplyOptions) Complete(f cmdutil.Factory, flags cmdutil.AdditionalFlags) error {
+	if flags.DryRunStrategyResult.Err != nil {
```

Comment:
It seems unnecessary to pass in the error here. We should just handle the error before calling the `Complete` function.

Reply:
Done